charts/scalardb/templates/scalardb/deployment.yaml

```diff
@@ -63,8 +81,8 @@ spec:
             timeoutSeconds: 1
           volumeMounts:
             - name: scalardb-config-file-volume
-              mountPath: /scalardb/server/database.properties
-              subPath: database.properties
+              mountPath: /scalardb/server/database.properties.tmpl
+              subPath: database.properties.tmpl
```

Comment:
For `dockerize` command runs in the Scalar DB Dockerfile, we need to mount properties file as `database.properties.tmpl`.